src/main/resources/SnakemakeBundle.properties

```diff
@@ -31,6 +31,9 @@ INSP.NAME.subworkflow.multiple.args=Only one argument is allowed for subworkflow
 # SmkSubworkflowRedeclarationInspection
 INSP.NAME.subworkflow.redeclaration=Only last subworkflow with the same name will be executed
 
+# SmkUndefinedNameInspection
+INSP.NAME.undefined.name=Undefined name
```

Comment:
Not very clear. The rule could be defined below, why we warn user that it is `undefined`?